docs/develop/feel-built-in-functions.md

```diff
@@ -135,6 +135,28 @@ not(true)
 // false
 ```
 
+### is defined()
+
+Checks if a given value is defined or not. Can be used to check if a variable, or a property of an object exists.
```

Comment:
The explanation here could be a little more detailed; especially the subtle difference between checking for null and existence/non-existence. Also the examples below could explicitly state that it will return true if the variable exists.